kura/distrib/src/main/resources/common/customize_snapshot.py

```diff
@@ -90,12 +90,12 @@ def main():
         
         interfaces_list = "lo"
     
-        for i, eth_name in enumerate(eth_names[:2]):
+        for i, eth_name in enumerate(eth_names):
             snapshot_content = snapshot_content.replace('ETH_INTERFACE_' + str(i), eth_name)
```

Comment:
I would try to keep this more explicit since, in the future, we don't know whether the template will change. Therefore I would suggest something like:

```suggestion
        for i, eth_name in enumerate(eth_names):
            if i < 2: # only replace the template content with the first two network interfaces
                snapshot_content = snapshot_content.replace('ETH_INTERFACE_' + str(i), eth_name)
```